NightscoutServiceKit/RemoteCommands/V1/Notifications/OverrideCancelRemoteNotification.swift

```diff
@@ -16,23 +16,24 @@ public struct OverrideCancelRemoteNotification: RemoteNotification, Codable {
     public let sentAt: Date?
     public let cancelOverride: String
     public let enteredBy: String?
```

Comment:
Added the otp string to other notification types, although they are not set on Nightscout/Caregiver currently. They will just be ignored as they are not required.